mappings/net/minecraft/entity/mob/EndermanEntity.mapping

```diff
@@ -18,6 +18,10 @@ CLASS net/minecraft/unmapped/C_qujqqtsx net/minecraft/entity/mob/EndermanEntity
 	METHOD m_dccostma playAngrySound ()V
 	METHOD m_jfjftevp isAngry ()Z
 	METHOD m_qkjqvawe isProvoked ()Z
+	METHOD m_qxprdwna hurtWithCleanWater (Lnet/minecraft/unmapped/C_sbxfkpyv;Lnet/minecraft/unmapped/C_nmolkgnw;F)Z
```

Comment:
what does "clean" mean

Reply:
Not sure, I meant to remove it from the mojmap name

Reply:
Yarn named it `damageFromPotion`, so probably this is used when a water bottle is thrown at an enderman

Reply:
Would you like me to change it? `hurtWithPlainPotion` or something?

Reply:
```suggestion
	METHOD m_qxprdwna hurtWithWaterPotion (Lnet/minecraft/unmapped/C_sbxfkpyv;Lnet/minecraft/unmapped/C_nmolkgnw;F)Z
```